src/group/mls_group/test_mls_group.rs

```diff
@@ -37,3 +41,84 @@ fn test_mls_group_persistence() {
 
     assert_eq!(alice_group, alice_group_deserialized);
 }
+
+#[test]
+fn test_failed_groupinfo_decryption() {
+    for version in Config::supported_versions() {
+        for ciphersuite in Config::supported_ciphersuites() {
+            let epoch = GroupEpoch(123);
+            let group_id = GroupId::random();
+            let tree_hash = vec![1, 2, 3, 4, 5, 6, 7, 8, 9];
+            let confirmed_transcript_hash = vec![1, 1, 1];
+            let extensions = Vec::new();
+            let confirmation_tag = vec![6, 6, 6];
```

Comment:
🤘

Reply:
I didn't even notice 😄. Credit goes to @raphaelrobert whom (I think) hid that in an existing test that I copy-pasted.